README.md

```diff
@@ -67,7 +76,7 @@ The following optional config parameters are available for the Odrive:
       "namespace": "rdk",
       "attributes": {
         "canbus_node_id": 0,
-        "odrive_config_file": "local/path/to/motor/config.json"
+        "connection_type": "canbus"
```

Comment:
[nit] did this change because of a rebase?

Reply:
no I just added it because I thought `connection_type` was a required config attribute but I just looked back at the scope and it's not? don't know if I made that up but I'll change it. And I removed `odrive_config_file` from the example to emphasize how optional it is